### ISSUE TITLE
Showing temperature in icon

### ISSUE THREAD

**Issue by MagicalDrizzle** [Shatur/optimus-manager-qt#100]
**Describe the problem or limitation you are having**

Would it be possible to add a temperature indicator in the tray icon? Maybe piping the output of nvidia-smi there?
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Shatur** [comment]
Yes, it should be possible. The hardest part is to draw a number on the tray icon. The approach is similar to displaying the counter of unread messages in Telegram Desktop.
